src/commands/enroll-data/GetOverallEnroll.ts

```diff
@@ -24,12 +24,14 @@ export default class GetOverallEnroll extends BaseCommand {
      * @inheritDoc
      */
     public async run(ctx: ICommandContext): Promise<number> {
-        const term = ctx.interaction.options.getString("term", true);
         const code = ctx.interaction.options.getString("course_subj_num", true);
+        const term = ctx.interaction.options.getString("term", false) ?? "all";
```

Comment:
why not just leave it as string or null? Then check if the term is null later on